examples/gce-acme/main.tf

```diff
@@ -15,104 +15,106 @@
  */
 
 provider "google" {
-  region = "${var.region}"
+  region = var.region
```

Comment:
The region should only be passed to modules and resources so let's just drop it from here.